docs/api-references/docs.md

```diff
@@ -6606,7 +6606,7 @@ string
 <td>
 <em>(Optional)</em>
 <p>Image used to tail slow log and set kernel parameters if necessary, must have <code>tail</code> and <code>sysctl</code> installed
-Optional: Defaults to busybox:1.26.2</p>
+Optional: Defaults to busybox:1.26.2. Recommended to set to 1.34.1 for new installations.</p>
```

Comment:
This file is generated automatically, please update the description in types.go, thanks!

Reply:
@dveeden Please check this comment since it will cause CI failure, thanks!

Reply:
I've ran `hack/update-all.sh`, the result is in c20a731daaa543dc1b90965cf4d59f6fe15a58a5

For this I had to modify my `go.sum` like this:
```patch
diff --git a/go.sum b/go.sum
index 5d4d2fdd..6308383b 100644
--- a/go.sum
+++ b/go.sum
@@ -80,7 +80,7 @@ github.com/PuerkitoBio/purell v1.1.1/go.mod h1:c11w/QuzBsJSee3cPx9rAFu61PvFxuPbt
 github.com/PuerkitoBio/urlesc v0.0.0-20160726150825-5bd2802263f2/go.mod h1:uGdkoq3SwY9Y+13GIhn11/XLaGBb4BfwItxLd5jeuXE=
 github.com/PuerkitoBio/urlesc v0.0.0-20170810143723-de5bf2ad4578 h1:d+Bc7a5rLufV/sSk/8dngufqelfh6jnri85riMAaF/M=
 github.com/PuerkitoBio/urlesc v0.0.0-20170810143723-de5bf2ad4578/go.mod h1:uGdkoq3SwY9Y+13GIhn11/XLaGBb4BfwItxLd5jeuXE=
-github.com/agiledragon/gomonkey/v2 v2.2.0 h1:QJWqpdEhGV/JJy70sZ/LDnhbSlMrqHAWHcNOjz1kyuI=
+github.com/agiledragon/gomonkey/v2 v2.2.0 h1:SXUF5UN7ZH67ORjFK+e1FoFRZczCSVUvwTywNY2+fcg=
 github.com/agiledragon/gomonkey/v2 v2.2.0/go.mod h1:ap1AmDzcVOAz1YpeJ3TCzIgstoaWLA6jbbgxfB4w2iY=
 github.com/agnivade/levenshtein v1.0.1/go.mod h1:CURSv5d9Uaml+FovSIICkLbAUZ9S4RqaHDIsdSBg7lM=
 github.com/ajg/form v0.0.0-20160822230020-523a5da1a92f/go.mod h1:uL1WgH+h2mgNtvBq0339dVnzXdBETtL2LeUXaIv25UY=
```
I haven't checked in the `go.sum` changes.

Without this it complained like this:
```
[dvaneeden@dve-carbon tidb-operator]$ ./hack/update-all.sh 
warning: ignoring symlink /home/dvaneeden/dev/pingcap/tidb-operator/deploy/gcp/charts/tidb-cluster
warning: ignoring symlink /home/dvaneeden/dev/pingcap/tidb-operator/deploy/gcp/charts/tidb-operator
warning: ignoring symlink /home/dvaneeden/dev/pingcap/tidb-operator/deploy/gcp/manifests/gke
go: downloading cloud.google.com/go v0.51.0
go: downloading github.com/robfig/cron v1.1.0
go: downloading go.etcd.io/etcd v0.5.0-alpha.5.0.20200819165624-17cef6e3e9d5
go: downloading github.com/agiledragon/gomonkey/v2 v2.2.0
go: downloading golang.org/x/sync v0.0.0-20201207232520-09787c993a3a
go: downloading github.com/pingcap/tidb v2.1.0-beta+incompatible
verifying github.com/agiledragon/gomonkey/v2@v2.2.0: checksum mismatch
	downloaded: h1:SXUF5UN7ZH67ORjFK+e1FoFRZczCSVUvwTywNY2+fcg=
	go.sum:     h1:QJWqpdEhGV/JJy70sZ/LDnhbSlMrqHAWHcNOjz1kyuI=

SECURITY ERROR
This download does NOT match an earlier download recorded in go.sum.
The bits may have been replaced on the origin server, or an attacker may
have intercepted the download attempt.

For more information, see 'go help module-auth'.
```

This could be related to goproxy etc.